applications/ConstitutiveLawsApplication/constitutive_laws_application.cpp

```diff
@@ -477,5 +479,32 @@ void KratosConstitutiveLawsApplication::Register()
     KRATOS_REGISTER_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS(ISOTROPIC_ANISOTROPIC_YIELD_RATIO);
     KRATOS_REGISTER_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS(ORTHOTROPIC_ELASTIC_CONSTANTS);
 
+    // For printing pursposes in Rule of Mixtures...
+    KRATOS_REGISTER_VARIABLE(CAUCHY_STRESS_VECTOR_LAYER_1)
```

Comment:
Huuuuuum, I don't like this... Huuuum, thinking alternatives

Reply:
Maybe we can add a macro to add several variables at once, the declaration time will be the same but at least the code will be cleaner

Reply:
I am still not happy with this...

Reply:
@pooyan-dadvand any idea how to solve it?